plugin/trino-opa/src/test/java/io/trino/plugin/opa/TestOpaAccessControlSystem.java

```diff
@@ -145,22 +124,24 @@ class BatchedAuthorizerTests
         public void setupTrino()
                 throws Exception
         {
-            setupTrinoWithOpa("v1/data/trino/allow", Optional.of("v1/data/trino/batchAllow"));
+            setupTrinoWithOpa(new OpaConfig()
+                                      .setOpaUri(OPA_CONTAINER.getOpaUriForPolicyPath(OPA_ALLOW_POLICY_NAME))
+                                      .setOpaBatchUri(OPA_CONTAINER.getOpaUriForPolicyPath(OPA_BATCH_ALLOW_POLICY_NAME)));
```

Comment:
Formatting seems off here.  This should be indented 8 spaces from the baseline... this looks like 30ish spaces